modules/kafka/kafka-dest-driver.c

```diff
@@ -512,6 +516,23 @@ _init_topic_name(KafkaDestDriver *self)
 }
 
 
+static gboolean
+_transaction_init(KafkaDestDriver *self)
+{
+  rd_kafka_error_t *error = rd_kafka_init_transactions(self->kafka, -1);
```

Comment:
I've found this in the documentation of librdkafka:
> Upon successful return from this function the application has to perform at least one of the following operations within transaction.timeout.ms to avoid timing out the transaction on the broker: ...

I guess this means that if there's no message at startup for transaction.timeout.ms (default 60 seconds), we will have failure on the first message producing.

Reply:
I've checked the previous version of this PR and although the documentation states that if no produce/commit/abort happens in `transaction.timeout.ms` time the transaction will fail, I didn't experience that.

Nevertheless, I agree with the new way of initializing the transaction.